.github/CHANGELOG.md

```diff
@@ -28,22 +28,9 @@
 1. [Systems] Pitot heating should now be active while engines are running - @lousybyte (lousybyte)
 1. [TEXTURES] Higher resoloution Mainpanel, fixed glareshield texture error - @Pleasure0102 (Pleasure)
 1. [ECAM] Overhauled WHEEL Page - @Benjozork (Benjamin Dupont)
-1. [CDU] Add approach waypoints to DCT page - @lengyc0208 (lyc)
 1. [Sounds] Added 3D emitters for all sounds
 1. [Sounds] Added sounds for Evac horn, emer cabin call, mech horn, gear extend/retract, and cargo door - @hotshotp (Boris)
-1. [CDU] Revised blue, green and amber colors on CDU - @lucky38i (Lucky38i)
-1. [CDU] Revised INIT A & B Page - @lucky38i (Lucky38i)
-1. [CDU] Fixed certain STARs not showing on arrivals page - @tyler58546 (tyler58546)
-1. [CHECKLISTS] Added several situations from QRH to checklists - FBI#7771
-1. [ECAM] Fix ECAM wheel page being selected before all gears are downlocked - @lukecologne (luke)
-1. [MISC] Added standby instrument brightness adjustment and attitude reset - @2hwk (2Cas#1022 on discord)
-1. [MISC] Added standby instrument bugs page and corrected standby instrument font and behaviour on power loss - @2hwk (2Cas#1022 on discord)
-1. [PFD] Added the ability to display metric altitudes - @lousybyte (lousybyte)
-1. [ECAM] Added messages related to anti-skid / N.W. steering being off - @Benjozork (Benjamin Dupont)
-1. [CDU] Updated HOLD Page to honeywell spec, PLAN page updated to show holds - @sepiroth887 (sepiroth887)
-1. [FCU] Fixed incrementing the QNH while in hPa mode - @lousybyte (lousybyte)
-1. [CDU] Follow Green Dot speed on approach with flaps clean - @veikkos (Veikko Soininen)
-1. [ND] Fixed cyan heading bug not showing on ND in VOR and LS modes - @AdenFlorian (David Valachovic)
+1. [PFD] Correct PFD attitude indicator onground/inflight @MMontalto (PiCcy)
```

Comment:
changelog merge did not go correctly

Reply:
I reverted to the master and added my own comment, correct?
Thanks, I'm sorry